src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -132,14 +132,14 @@ protected StorageEnvironmentOptions(string tempPath)
 
             ShouldUseKeyPrefix = name => false;
 
-            MaxLogFileSize = ((sizeof(int) == IntPtr.Size ? 32 : 256)*Constants.Size.Megabyte);
```

Comment:
Those settings will never actually take effect. We are calling this in the ctor, before the user has a change to change it.
Need to actually change those values by default in the setter.